eth/backend.go

```diff
@@ -395,25 +420,25 @@ func (s *Ethereum) BLSbase() (eb common.Address, err error) {
 		return blsbase, nil
 	}
 
-	return s.Etherbase()
+	return s.Validator()
 }
 
 // isLocalBlock checks whether the specified block is mined
 // by local miner accounts.
 //
-// We regard two types of accounts as local miner account: etherbase
-// and accounts specified via `txpool.locals` flag.
+// We regard two types of accounts as local miner account: the validator
+// address and accounts specified via `txpool.locals` flag.
 func (s *Ethereum) isLocalBlock(block *types.Block) bool {
 	author, err := s.engine.Author(block.Header())
 	if err != nil {
 		log.Warn("Failed to retrieve block author", "number", block.NumberU64(), "hash", block.Hash(), "err", err)
 		return false
 	}
-	// Check whether the given address is etherbase.
+	// Check whether the given address is configured validator.
 	s.lock.RLock()
-	etherbase := s.etherbase
+	validator := s.validator
```

Comment:
Question for reviewer: is this correct? I assume "local miner account" in this context refers to the message signer, but maybe I'm wrong.

Reply:
Looks like what you did is correct.  The `s.engine.Author` function retrieves the signer of the block header:  

https://github.com/celo-org/celo-blockchain/blob/master/consensus/istanbul/backend/engine.go#L109

https://github.com/celo-org/celo-blockchain/blob/master/consensus/istanbul/backend/engine.go#L900

Reply:
Perhaps renaming the variable `author` to `signer` will make it a bit more clear?